tests/python/gaia-ui-tests/gaiatest/apps/bzlite/regions/create_bug.py

```diff
@@ -0,0 +1,33 @@
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+
+class CreateBug(Base):
+
+    _title_input = (By.ID, 'summary')
+    _description_input = (By.ID, 'description')
+    _thumbnail_photo_locator = (By.CSS_SELECTOR, ".btn-file")
+    _button_submit = (By.ID, 'submit')
+
+
+    def _fill_title(self,title):
+        title_element = self.marionette.find_element(*self._title_input)
+        title_element.send_keys(title)
+
+    def _fill_description(self,description):
+        description_element = self.marionette.find_element(*self._description_input)
+        description_element.send_keys(description)
+
+    def _fill_picture(self):
```

Comment:
We can rename this function `fill_picture_with_first_image_in_gallery` as this is what it does.